### PR TITLE
Host in lms

### PR DESCRIPTION
**Pull request by apalaniuk**
@ryantmer Feel like reviewing a micro-PR?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,17 @@ The demo app can be visited at:
 
 The My Courses widget pulls data from an instance of the User Info Service to get the enrollments for a user. This functionality is currently in a beta-ish state, as doing so more cleanly would require the widget to be running hosted on the LMS. As it stands now, some changes are required to the LMS to enable CORS headers on all requests for this approach to work.
 
+## Local Testing
+
+Testing from within LMS:
+
+1) Checkout brightspace/my-courses-widget and brightspace/brightspace-integration projects into same folder
```
Comment (d2l-joelp) [inline README.md]: I was able to use `bower link` during development. Not sure how checking out into the same folder would work either, unless I'm misunderstanding the instructions.

File: d2l-my-courses.html [hunk 2]
```diff
@@ -77,11 +77,11 @@
 			},
 			uisEndpoint: {
```
Comment (d2l-joelp) [inline d2l-my-courses.html]: In a PR I have coming, I'm getting rid of both `uisEndpoint` and `lms` properties and using only relative paths. But as it stands it doesn't hurt anything to make these changes. Just thought I'd let you know.

File: README.md [hunk 3]
```diff
@@ -27,6 +27,17 @@ The demo app can be visited at:
 
 The My Courses widget pulls data from an instance of the User Info Service to get the enrollments for a user. This functionality is currently in a beta-ish state, as doing so more cleanly would require the widget to be running hosted on the LMS. As it stands now, some changes are required to the LMS to enable CORS headers on all requests for this approach to work.
```
Comment (ryantmer) [inline README.md]: This should probably be updated. I guess not strictly part of this change, but this change does take it one step further from truth, so might as well :)

File: d2l-my-courses.html [hunk 4]
```diff
@@ -77,11 +77,11 @@
...
 				type: String,
-				value: 'http://127.0.0.1:3000/enrollments'
+				value: 'https://pulse-user-info.api.brightspace.com/enrollments'
 			},
 			lms: {
 				type: String,
-				value: 'http://127.0.0.1:44444'
+				value: ''
```
Comment (ryantmer) [inline d2l-my-courses.html]: Lack of default value sounds good, but might want to make note of this in the readme that if you don't specify it, it won't work and will just spew errors in the console

**Comment by ryantmer** [comment]
:+1: :shipit:

File: README.md (hunk 1 above)
Comment (apalaniuk) [inline README.md]: This is probably a much better idea and only not done out of sheer ignorance. I'll test and update.

**Pull request merged**